2.0/AudioSimple/glTF/AudioSimple.gltf

```diff
@@ -0,0 +1,381 @@
+{
+  "asset": {
+    "generator": "Khronos glTF Blender I/O v1.8.19",
+    "version": "2.0"
+  },
+  "extensionsUsed": ["KHR_audio"],
+  "extensions": {
+    "KHR_audio": {
+      "emitters": [
+        {
+            "name": "Global Emitter",
+            "type": "global",
+            "gain": 0.5,
+            "sources": [0]
+          },
+        {
+          "name": "Positional Emitter",
+          "type": "positional",
+          "gain": 1,
+          "sources": [1],
+          "positional": {
+            "coneInnerAngle": 6.283185307179586,
+            "coneOuterAngle": 6.283185307179586,
+            "coneOuterGain": 0.0,
+            "distanceModel": "inverse",
+            "maxDistance": 10.0,
+            "refDistance": 1.0,
+            "rolloffFactor": 0.8
+          }
+        }
+      ],
+      "sources": [
+        {
+          "name": "Ambient Audio Source",
+          "gain": 1,
+          "autoPlay": true,
+          "loop": true,
+          "audio": 0
+        },
+        {
+          "name": "Talking Audio Source",
+          "gain": 1,
+          "autoPlay": true,
+          "loop": true,
+          "audio": 1
+        }
+      ],
+      "audio": [
+        {
+          "uri": "ambient.mp3"
+        },
+        {
+          "uri": "talking.mp3"
+        }
+      ]
+    }
+  },
+  "scene": 0,
+  "scenes": [
+    {
+      "name": "Scene",
+      "nodes": [0, 2],
+      "extensions": {
+        "KHR_audio": {
+            "emitters": [
+                0
+            ]
+        }
+      }
+    }
+  ],
+  "nodes": [
+    {
+      "mesh": 0,
+      "name": "Plane",
+      "scale": [10, 10, 10]
+    },
+    {
+      "mesh": 1,
+      "name": "Sphere",
+      "scale": [0.5, 0.5, 0.5],
+      "translation": [5.605831146240234, 1.0160478353500366, 0],
+      "extensions": {
+        "KHR_audio": {
+            "emitter": 1
```

Comment:
This node has both an audio emitter and a Mesh. This significantly complicates import into Godot Engine which only allows one type per node, since we would need to import 2 nodes in this case.

Instead, I would be in favor of modifying the KHR_audio standard to only allow audio emitters on nodes that aren't meshes. Even aside from implementation difficulty, it makes more sense to have these be separate, so that an object's audio transform is not necessarily locked to the mesh transform.